CODE_OF_CONDUCT.md

```diff
@@ -1,43 +1,82 @@
 ## CNCF Community Code of Conduct v1.0
```

Comment:
looks like the CNCF Code of Conduct was not updated yes? https://github.com/cncf/foundation/blob/master/code-of-conduct.md

> This Code of Conduct is adapted from the Contributor Covenant (http://contributor-covenant.org), version 1.2.0, available at http://contributor-covenant.org/version/1/2/0/

Should probably be updated there